pkg/transform/jq_common.go

```diff
@@ -99,16 +115,32 @@ func parseTimeLayout(params []any) (string, error) {
 	} else if len(params) == 1 {
 		str, ok := params[0].(string)
 		if !ok {
-			return "", fmt.Errorf("Function argument is invalid '%v'; expected string", params[0])
+			return "", fmt.Errorf("function argument is invalid '%v'; expected string", params[0])
 		}
 		return str, nil
 	} else {
-		return "", fmt.Errorf("Too many function arguments - %d; expected 1", len(params))
+		return "", fmt.Errorf("too many function arguments - %d; expected 1", len(params))
+	}
+}
+
+func resolveHash(input any, params []any) (string, error) {
```

Comment:
@oo-sleeper and just for completenes - this one I'm leaving as-is, because it looks like it does actually resolve the hash, rahter than return a function to do so.